lua/telescope/_extensions/attempt.lua

```diff
@@ -5,7 +5,7 @@ local action_state = require 'telescope.actions.state'
 local action_set = require 'telescope.actions.set'
 
 local function entry_to_filename(e)
-  return e.value.filename .. (e.value.ext ~= '' and ('.' .. e.value.ext) or '')
+  return e.value.filename .. (e.value.ext and ('.' .. e.value.ext) or '')
```

Comment:
The issue here is that `''` is actually truthy in lua. You would need to use both conditions

Reply:
Oh, duh. Thanks, will get this fixed up.

Reply:
Fixed and tested!

![image](https://user-images.githubusercontent.com/1082451/168497448-c36d1d27-bbfe-485f-aa21-237b4ee903c2.png)